Pod/Classes/Optional Data Sources/VOKPagingFetchedResultsDataSource.m

```diff
@@ -52,21 +55,27 @@ - (void)setupAccessoryViews
 {
     //Attach given views, or generate default views.
     if (!self.headerView) {
-        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:(CGRect){0, -30, self.tableView.frame.size.width, 30}];
+        CGRect headerFrame = (CGRect){0, -DefaultAccessoryHeight, self.tableView.frame.size.width, DefaultAccessoryHeight};
+        self.headerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:headerFrame];
     }
     
-    [self.headerView setFrame:(CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size}];
+    self.headerView.frame = (CGRect){0, -self.headerView.frame.size.height, self.headerView.frame.size};
     [self.tableView addSubview:self.headerView];
     
     if (!self.footerView) {
-        self.footerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:(CGRect){0,
+        CGRect footerFrame = (CGRect){0,
             MAX(self.tableView.contentSize.height, self.tableView.bounds.size.height),
-            self.tableView.frame.size.width, 30}];
+            self.tableView.frame.size.width,
+            DefaultAccessoryHeight};
+        self.footerView = [[VOKDefaultPagingAccessory alloc] initWithFrame:footerFrame];
     }
     
     [self.tableView addSubview:self.footerView];
     
-    [self.tableView addObserver:self forKeyPath:@"contentSize" options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld | NSKeyValueObservingOptionPrior context:NULL];
+    [self.tableView addObserver:self
+                     forKeyPath:VOKKeyForInstanceOf(UITableView, contentSize)
+                        options:NSKeyValueObservingOptionNew | NSKeyValueObservingOptionOld | NSKeyValueObservingOptionPrior
```

Comment:
`NSKeyValueObservingOptionPrior` causes the notification to be sent both prior to the change and after the change, but it looks like we're carrying out the same action both times.  Should we either not be using `NSKeyValueObservingOptionPrior` or checking the value in the change dictionary for the key `NSKeyValueChangeNotificationIsPriorKey`?